views/components/ConfigurationsList/index.jsx

```diff
@@ -305,15 +305,32 @@ const ConfigurationsList = ({ user, classes, theme }) => {
         }
         return response.json()
       })
-      .then((response) => {
+      .then(({ data }) => {
         setState((prevState) => {
           return {
             ...prevState,
-            preferences: babyProofPreferences(response),
+            preferences: babyProofPreferences(data.preferences),
           }
         })
       })
   }
+  const removeConfiguration = async (_id) => {
+    return window
+      .fetch(apiRoutes.configurations.userConfiguration(user.uid, _id), {
```

Comment:
do we need to nest this route within `configurations`? It seems like `apiRoutes.userConfigurations` is a good top-level route. Also, why are we prefixing the `id` with an `_`?

```suggestion
      .fetch(apiRoutes.userConfiguration(user.uid, _id), {
```

Reply:
I am confused, we discussed about nesting these routes